src/main/java/com/gbtec/gameoflife/WebSocketConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.gbtec.gameoflife;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker;
+import org.springframework.web.socket.config.annotation.StompEndpointRegistry;
+import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
+
+@Configuration
+@EnableWebSocketMessageBroker
+public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
+
+    @Override
+    public void configureMessageBroker(MessageBrokerRegistry config) {
+        config.enableSimpleBroker("/gol");
```

Comment:
Do not abbreviate `gol`. To be most simple, we could just remove `/gol`, as the whole package and project is named `gameoflife`, anyway.